docs/index.bs

```diff
@@ -1576,11 +1576,16 @@ spec: storage; urlPrefix: https://storage.spec.whatwg.org/
       The <dfn method for="InstallEvent"><code>addRoutes(|rules|)</code></dfn> method steps are:
 
         1. Let |routerRules| be a list of {{RouterRule}} dictionaries.
+        1. Let |hasFetchEventSource| be a boolean, initially false.
+        1. Let |serviceWorker| be the [=current global object=]'s associated [=ServiceWorkerGlobalScope/service worker=].
+        1. If |serviceWorker|'s [=list of router rules=] [=list/contains=] a {{RouterRule}} whose {{RouterRule/source}} is "{{RouterSource/fetch-event}}", set |hasFetchEventSource| to true.
```

Comment:
Around this line, we might need to set |serviceWorker|'s [=list of router rules=] to |routerRules|?
Otherwise, previous router rules can be cleared in L1588?

Reply:
Set `|routerRules|` as the expected behavior of `addRoutes()`

Reply:
Thank you.
nit but you can use |routerRules| instead of saying "|serviceWorker|'s [=list of router rules=]" again in L1581.

Reply:
Updated. I realized that `|hasFetchEventSource|` is not necessary, so removed it.